docs/index.html

```diff
@@ -372,9 +371,10 @@ <h2 id="blur">Gaussian Blur</h2>
             relative to blur radius.
         <p>
             This approach was implemented in Pillow <code>2.7.0</code>.
-            According to the tests, even if other convolution implementations
-            are superior to box filters' approximation with small sigma,
-            this benefit disappears alongside with a larger sigma.
+            According to the tests, even if box filters' approximation
+            could be little slower with a small sigma,
+            with a larger sigma this gap disappears and 
+            box filters' approximation becomes much faster.
```

Comment:
Are we talking about a single box filter or a group filters known as "box filters"? If the latter, then this is fine. If the former, then "box filters'" should probably be changed to "box filter's". Otherwise changes look good.